bin/yaml/swift.yaml

```diff
@@ -31,6 +31,17 @@ compilers:
           - '5.4'
           - '5.5'
           - '5.6'
+    5.7-nightly:
+      if: nightly
+      type: restQueryTarballs
+      install_always: true
+      url: https://swift.org/builds/swift-5.7-branch/ubuntu2004/latest-build.yml
```

Comment:
Oh wow! Top marks for working out how to do that in my shonky configuration language!

Reply:
Hehe, it's not so shonky in comparison to others I've encountered 😂